backend/backend/settings/dev.py

```diff
@@ -8,6 +8,14 @@
 DEBUG = True
 # LEDGER_CALL_RETRY = False  # uncomment to overwrite the ledger setting value
 
+# Enable Browsable API
+REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] = REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] + (
```

Comment:
```suggestion
REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] += (
```
Should work but not tested.

Reply:
👍